pretyping/coercion.ml

```diff
@@ -36,7 +36,7 @@ let _ =
   Goptions.(declare_bool_option
     { optdepr  = false;
       optname  = "use typeclass resolution during conversion";
-      optkey   = ["Typeclass"; "Resolution"; "For"; "Conversion"];
+      optkey   = ["Typeclasses"; "Resolution"; "For"; "Conversion"];
```

Comment:
Really?  I thought it's always "typeclass resolution" (to refer to the resolution used by all typeclasses), in the same way that one might say "gamer culture" to refer to the culture of all gamers.

Reply:
I made it match the already existing documentation and the other typeclass options.
It's a component prefix, like `Typeclasses.resolution_for_conversion`.